README.md

```diff
@@ -29,6 +29,8 @@ The goal is to **manage commit hooks with Nix** and solve the following:
     in {
       pre-commit-check = nix-pre-commit-hooks.run {
         src = ./.;
+        # Leaving out this option falls back to install all hooks into the 'commit' stage
+        default_stages = ["manual" "push"];
```

Comment:
We should stick to pre-commit defaults by default in our examples, to minimize surprises. I don't think we should suggest this to novices if they're only running some quick formatters.

```suggestion
        # If your hooks are intrusive, avoid running on each commit with a default_states like this:
        # default_stages = ["manual" "push"];
```

Also avoids a negation :)

Reply:
I agree with Robert, we shouldn't modify the default of pre-commit project.